src/main/java/com/xebialabs/xlrelease/ci/server/XLReleaseConnectorImpl.java

```diff
@@ -32,9 +36,27 @@ public ClientResponse getVariablesResponse(final String templateId) {
     @Override
     public ClientResponse createReleaseResponse(final String resolvedTemplate, final String resolvedVersion, final List<NameValuePair> variables) {
         WebResource service = buildWebResource();
+        String queryString = resolvedTemplate.replaceAll(SLASH_ESCAPE_SEQ,SLASH_MARKER);
```

Comment:
Lets move string manipulation operations for folders and template to common place